docs/SapEcc-batchsource.md

```diff
@@ -0,0 +1,51 @@
+# SAP OData Batch Source
+
+Description
+-----------
+This plugin reads data from SAP OData service.
```

Comment:
The design had some information about how to setup the OData service, which seems like it would be useful to include here.

Reply:
The following info is added:
```
OData(Open Data Protocol) is an open data access protocol from Microsoft that allows the creation and consumption of
query-able and interoperable RESTful APIs in a simple and standard way. SAP Netweaver Gateway comes with design-time
tools to facilitate modeling OData services for consumption, particularly [SAP Gateway Service Builder]
(transaction SEGW). For detailed information about the end-to-end service development process, see
[Getting Started with the Service Builder].
```